templates/compositions/ec-component-link-block/link-block.html.twig

```diff
@@ -24,6 +24,7 @@
 {% set _links = links|default([]) %}
 {% set _css_class = 'ecl-link-block' %}
 {% set _extra_attributes = '' %}
+{% set _color = ecl_component_library == 'ec' ? 'neutral-20' : 'primary-5' %} %}
```

Comment:
Why here use `neutral-20` color? Do we have any reference on what exactly color should be used?

Reply:
To be honest we can discuss and decide I think, I chose that one pretty much at random. But I will use the twig method created by @22Alexandra so it is centralized if we want to change it in the future